src/App.js

```diff
@@ -181,23 +191,63 @@ const App = () => {
 		tools.push({name: name, icon: icon})
 	}
 
+	const login = () => {
+		fetch("https://skilling-59c90.firebaseio.com/anthony.json")
+      .then(res => res.json())
+      .then(
+        (result) => {
+					updateLevel("wood", woodcutting, setWoodcutting, result.woodXp)
+					updateLevel("stone", quarrying, setQuarrying, result.stoneXp)
+					updateLevel("fish", fishing, setFishing, result.fishingXp)
+					updateLevel("plank", makePlank, setMakePlank, result.plankXp)
+					updateLevel("fire", makeCampfire, setMakeCampfire, result.fireXp)
+					updateLevel("cook", cooking, setCooking, result.cookXp)
+					updateLevel("prospect", prospecting, setProspecting, result.prospectXp)
+					updateLevel("tool", toolMaking, setToolMaking, result.toolXp)
+					updateLevel("sacrifice", sacrifice, setSacrifice, result.sacrificeXp)
+        },
+        (error) => {
+          this.setState({
+            isLoaded: true,
+            error
+          });
```

Comment:
`  this.setState({
            isLoaded: true,
            error
          });`

I'm getting an error when I try to plug in my own Firebase URL and call the login function by clicking Load User.

Also, not sure what this bit of code is supposed to do. Could you explain?

Reply:
I have no idea, it was a copy/paste from a http example.

Does your Firebase db have public read/write?